components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -274,27 +238,28 @@ public void variantsSetup(Set<String> enabledFields) throws Exception
         String sectionName = "variants";
         String[] fieldIds =
             { "variants", "variants_protein", "variants_transcript", "variants_dbsnp", "variants_zygosity",
-            "variants_effect", "variants_interpretation", "variants_inheritance", "variants_evidence",
-            "variants_segregation", "variants_sanger", "variants_coordinates" };
+                "variants_effect", "variants_interpretation", "variants_inheritance", "variants_evidence",
```

Comment:
I used my Phenotips formatting tool before the PR, should I have only used it on the section of code I was working on?